nav2_costmap_2d/src/costmap_2d_ros.cpp

```diff
@@ -84,6 +98,15 @@ Costmap2DROS::Costmap2DROS(
     "nav2_costmap_2d::StaticLayer",
     "nav2_costmap_2d::ObstacleLayer",
     "nav2_costmap_2d::InflationLayer"}
+{
+  declare_parameter(
```

Comment:
Why is `plugins` singled out? That appears the same either way.

Reply:
Just didn't want to pass it to init() but I can

Reply:
Oops, you're right, fixed